src/test/regress/upgrade/config.py

```diff
@@ -0,0 +1,41 @@
+OLD_BINDIR = 'OLD_BINDIR'
+NEW_BINDIR = 'NEW_BINDIR'
+TEMP_DIR = 'TEMP_DIR'
+NEW_DATADIR = 'NEW_DATADIR'
+OLD_DATADIR = 'OLD_DATADIR'
+PG_SRCDIR = 'PG_SRCDIR'
+BEFORE_UPGRADE_SCHEDULE = './before_upgrade_schedule'
```

Comment:
for the normal tests we pass these in via arguments as well. Together with `EXTRA_TESTS='...'`.

This combination makes it easy to have simpler base schedules as well and some `EXTRA_TESTS` to run on top of the base schedule. This is really helpful when you are creating a new test, instead of running everything you can run a bare minimum and your new test without running the complete suite.

Did you have a look if we could have the same workflow here?